bin/diagnostic.js

```diff
@@ -1 +1,14 @@
 'use strict'
+const ask = require('../lib/ask.js')
+
+let answer = 'not empty'
+
+while (answer !== '') {
+  answer = ask('Guess my secret?')
+  if (answer >= 1 && answer <= 10) {
+    console.log('You guessed it!')
+  } else {
+    console.log('The guess needs to be between 1 and 10')
```

Comment:
You want to set a number here for the user to guess, in addition to checking to make sure their answer falls between 1-10